src/examples/rest/src/frontend/types.ts

```diff
@@ -21,179 +21,316 @@ export type Scalars = {
   JSON: { input: any; output: any; }
```

Comment:
It looks like the output of the file is no longer alphabetical. Is it possible to get that ordering back?

Reply:
I don't know but I agree that sorting would be preferable. I will see what options there are.

Reply:
Added "sort: true" to config